health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -392,15 +325,15 @@ export const UICustomizations = {
         wardCode: "wardCode[0].code",
         socialCategory: "socialCategory.code",
       };
-      const textConfig = ["name", "individualId"];
+      const textConfig = ["name", "individualId"]
       let Individual = Object.keys(requestBody)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'Individual' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)